client/package.json

```diff
@@ -72,6 +72,20 @@
                     "command": "checkstyle.checkCodeWithCheckstyle",
                     "when": "resourceExtname == .java"
                 }
+            ],
+            "explorer/context": [
+                {
+                    "command": "checkstyle.checkCodeWithCheckstyle",
+                    "when": "explorerResourceIsFolder == false && resourceExtname == .java",
```

Comment:
Is `explorerResourceIsFolder == false` implied in `resourceExtname == .java` ?

Reply:
Seems not.